apps/condo/domains/resident/schema/RegisterResidentInvoiceService.js

```diff
@@ -82,12 +89,40 @@ const RegisterResidentInvoiceService = new GQLCustomSchema('RegisterResidentInvo
                 const { dv, sender } = data
                 const userId = get(context, ['authedItem', 'id'])
 
+                if (data.invoiceRows.length === 0) {
+                    throw new GQLError(ERRORS.EMPTY_ROWS, context)
+                }
+
                 const locale = extractReqLocale(context.req) || conf.DEFAULT_LOCALE
 
                 const resident = await getByCondition('Resident', { deletedAt: null, id: data.resident.id })
 
+                const priceScopesCounts = {}
+                const priceScopesIds = data.invoiceRows.map((row) => {
+                    const id = get(row, ['priceScope', 'id'])
+                    set(priceScopesCounts, id, get(row, 'count', 0))
+
+                    return id
+                })
+
+                const priceScopes = await MarketPriceScope.getAll(context, { deletedAt: null, id_in: priceScopesIds })
+
+                if (priceScopes.length === 0) {
+                    throw new GQLError(ERRORS.EMPTY_ROWS, context)
+                }
+
+                const uniqueOrganizationIds = uniq(priceScopes
+                    .map(priceScope => get(priceScope, ['marketItemPrice', 'marketItem', 'organization', 'id']))
+                )
+
+                if (uniqueOrganizationIds.length > 1) {
```

Comment:
We have relatedOrganizations check in access. Seems, like we do not need this validation